csrc/executor.cpp

```diff
@@ -179,6 +179,79 @@ void FusionExecutor::debugCompileFusionFromStr(
       fusion_id_ > 0, "assign a fusion_id_ <= 0 is not accepted.");
 }
 
+static inline DataType ComputeIndexType(
```

Comment:
nit: `computeIndexType`. Also, it's preferred to use anonymous namespace functions over static functions.